Source: kloeri@exherbo.org
Upstream: No
Reason: Disable cpuclock2 test as it fails in kvm

--- rt/Makefile.orig	2009-06-01 12:28:44.000000000 +0000
+++ rt/Makefile	2009-06-01 12:28:56.000000000 +0000
@@ -47,7 +47,7 @@
 	 tst-mqueue1 tst-mqueue2 tst-mqueue3 tst-mqueue4 \
 	 tst-mqueue5 tst-mqueue6 tst-mqueue7 tst-mqueue8 tst-mqueue9 \
 	 tst-timer3 tst-timer4 tst-timer5 \
-	 tst-cpuclock1 tst-cpuclock2 \
+	 tst-cpuclock1 \
 	 tst-cputimer1 tst-cputimer2 tst-cputimer3 \
 	 tst-clock2
 
